PWGHF/TableProducer/trackIndexSkimCreator.cxx

```diff
@@ -3431,6 +3431,9 @@ struct HfTrackIndexSkimCreatorLfCascades {
     df3.setWeightedFinalPCA(useWeightedFinalPCA);
 
     uint8_t hfFlag = 0;
+    bool isGoogForXi2Prong = true;
+    bool isGoogForOmega2Prong = true;
+    bool isGoogForXi3Prong = true;
```

Comment:
There is no need of the 3-prong one; whenever a 3-prong candidate is not selected you can continue, because in any case the triplet will not be used anymore.